snowplow/iglu-client-embedded/schemas/com.metabase/csvupload/jsonschema/1-0-0

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "CSV upload events",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "csvupload",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "event": {
+      "description": "Event name",
+      "type": "string",
+      "enum": ["csv_upload_successful", "csv_upload_failed"],
+      "maxLength": 128
+    },
+    "model_id": {
+      "description": "Unique identifier for the newly created model",
+      "type": ["string", "null"],
+      "maxLength": 128
+    },
+    "upload_speed": {
```

Comment:
weird to call it's speed but the content is time it takes to update, how about `upload_time` or `upload_duration`?